src/Microsoft.Data.Sqlite/Utilities/NativeLibraryLoader.cs

```diff
@@ -103,6 +80,8 @@ public static bool IsLoaded(string dllName)
             return handle != IntPtr.Zero;
         }
 
+        private static bool IsDNX() => PlatformServices.Default != null;
```

Comment:
Won't this always be true with its [current implementation](https://github.com/aspnet/dnx/blob/0b4140fd6827acc912eed858431922574cd3f9f2/src/Microsoft.Extensions.PlatformAbstractions/PlatformServices.cs#L9-L17)?